### PR TITLE
Add arraylist randomizer

### PR DESCRIPTION
**Pull request by SpencerTaft**

### CODE REVIEW

**Comment by houtan1** [comment]
I'm going to run a *sanity check* on it

**Comment by houtan1** [comment]
Turns out, to test it, I'd need to merge it with a local branch, on my machine, and then run it, and then push it back up to GitHub. I'm going to trust that you've tested it and it works. I'll just **review** the code.

File: app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java [hunk 1]
```diff
@@ -7,6 +7,9 @@
 import android.widget.ListView;
 
 import java.util.ArrayList;
+import java.util.List;
+import java.util.Collections;
+import java.util.Random;
```
Comment (houtan1) [inline app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java]: wow, that's cool that it exists

File: app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java [hunk 2]
```diff
@@ -53,4 +59,14 @@ protected void onCreate(Bundle savedInstanceState) {
         //connect myListView with myAdapter
         myListView.setAdapter(myAdapter);
     }
+
+    /*
+    This function takes an input arraylist and then randomizes it
+    */
+    ArrayList<String> randomize_ArrayList(ArrayList<String> input_array)
```
Comment (houtan1) [inline app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java]: it's in its own function... aw yiss!

File: app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java [hunk 3]
```diff
@@ -53,4 +59,14 @@ protected void onCreate(Bundle savedInstanceState) {
...
+    {
+        long seed = System.nanoTime();
```
Comment (houtan1) [inline app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java]: based on system time so it doesn't do the same randomization every time, noice!

Comment (SpencerTaft) [inline app/src/main/java/com/example/houta/dadvice_v01/MainActivity.java]: noice!  Yeah it all runs, taking a look at your stuff now

**Pull request merged**